helpers/deploy/deploymentConfig.ts

```diff
@@ -160,6 +160,18 @@ export const config: Config = {
       { method: "setMaxDailyReceiveLimit(uint256)", args: [100] },
     ],
   },
+  opmainnet: {
+    methods: [
+      { method: "setMinDstGas(uint16,uint16,uint256)", args: [LZ_CHAINID["opmainnet"], 0, 200000] },
+      { method: "setMaxDailyReceiveLimit(uint256)", args: [100] },
+    ],
+  },
+  opsepolia: {
+    methods: [
+      { method: "setMinDstGas(uint16,uint16,uint256)", args: [LZ_CHAINID["opsepolia"], 0, 200000] },
```

Comment:
```suggestion
      { method: "setMinDstGas(uint16,uint16,uint256)", args: [LZ_CHAINID["sepolia"], 0, 200000] },
      { method: "setMinDstGas(uint16,uint16,uint256)", args: [LZ_CHAINID["bsctestnet"], 0, 200000] },
      { method: "setMinDstGas(uint16,uint16,uint256)", args: [LZ_CHAINID["opbnbtestnet"], 0, 200000] },
      { method: "setMinDstGas(uint16,uint16,uint256)", args: [LZ_CHAINID["arbitrumsepolia"], 0, 200000] },
```

Reply:
@GitGuru7 I have updated for opsepolia, please have a look here: 26146d8556611d524a03f663eda792ef668c91fd

Reply:
Mainnet is not deployed yet

Reply:
And here for bscmainnet only, @GitGuru7